doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,114 +3,103 @@
+* Improved layout for data type rows in the *Data Types* tab. You can now drag data type elements between parent data types.
+* Support for {CENTRAL} data objects as DMN data types. You can now import existing data objects in {CENTRAL} as DMN data types in the DMN designer. To import existing data objects as DMN data types, go to *Data types* -> *Import data object* in the DMN designer.
 
 For more information about DMN support in {PRODUCT} {PRODUCT_VERSION}, see the following resources:
 
 * {URL_DMN_MODELS}[_{DMN_MODELS}_]
 * {URL_GETTING_STARTED_DECISION_SERVICE}[_{GETTING_STARTED_DECISION_SERVICE}_]
 
-=== Ability to convert a guided decision table to a spreadsheet decision table
+=== High-availability {CENTRAL}
 
-You can convert a guided decision table in {CENTRAL} to an XLS spreadsheet decision table file for offline reference and file sharing. The guided decision table must be an extended entry guided decision table in order to be converted. The conversion tool does not support limited entry guided decision tables. To convert a guided decision table, navigate to the guided decision table asset that you want to convert and in the upper-right toolbar of the guided decision tables designer, click *Convert to XLS*.
+High-availability (HA) deployment of {CENTRAL} is now supported on premises (using {EAP}) and on {OPENSHIFT} version 3.11.
 
-WARNING: Guided decision tables and spreadsheet decision tables are different decision table formats that support different features. Any supported features that differ between the two decision table formats are modified or lost when you convert one decision table format to the other.
+In {PRODUCT} version 7.6, HA deployment of {CENTRAL} is supported with certain limitations:
 
-ifdef::PAM[]
-=== Bulk reassignment supported for tasks in {CENTRAL}
-
-In the *Task Inbox* and *Tasks* pages in {CENTRAL}, you can now perform bulk reassignment over multiple tasks in a single operation. To reassign tasks in bulk, select two or more tasks, click the *Bulk Actions* menu in the upper-right corner of the window, and select *Bulk Reassign*.
+* If a {CENTRAL} instance fails while a user works with it, the user can get an error message and then is redirected to another pod. If {CENTRAL} is configured to use Red Hat Single Sign-on, logging on again is not required. 
```

Comment:
If a {CENTRAL} instance fails while a user is working with it, the user can receive...

I would say ...while a user is using it, then the user might receive and error message. In this case, the user is directed...

I assume that the user does not always receive an error.